lib/open_api_parser/pointer.rb

```diff
@@ -1,5 +1,8 @@
 module OpenApiParser
+  # Responsible for interpreting the fragment portion of a $ref value
+  # as a JSON Pointer and resolving it within a given document.
```

Comment:
Would you mind removing comments for consistency with the rest of the library?

Reply:
Did you mean these ones: 9dcc0bd6c92c916d99ae6fa5a7b3838d8368220a
or including all the other code comments?

Reply:
All comments.

Reply:
Remove the remainder in 0989f6a